packages/monaco/src/typings/monaco/index.d.ts

```diff
@@ -15,7 +15,7 @@
  ********************************************************************************/
 
 /* eslint-disable @typescript-eslint/no-explicit-any */
-/// <reference types='@typefox/monaco-editor-core/monaco'/>
+/// <reference types='@theia/monaco-editor-core/monaco'/>
```

Comment:
@RomanNikitenko now, the tedious part is coming 😬 

Usually we go through this file and check that signatures exist and matching in https://github.com/theia-ide/vscode/tree/standalone/0.19.x If not they should be aligned

Reply:
@akosyakov 
Should we follow **strictly**  the corresponding signatures of vs code?
Or the answer is: it depends?

For example, `IBulkEditService` [in current](https://github.com/eclipse-theia/theia/blob/1eaecd5cbf93468a74988ddcd347402fcbf14cbe/packages/monaco/src/typings/monaco/index.d.ts#L32) `index.d.ts` differs  from [0.17 version](https://github.com/theia-ide/vscode/blob/63d87164d0bc8c6206d9339c195289c93665028e/src/vs/editor/browser/services/bulkEditService.ts#L23) of `IBulkEditService` 

0.17 version `IBulkEditService` contains:
- field `_serviceBrand`
- `apply` method has `options?: IBulkEditOptions` as optional parameter.

So, should we align it now for 0.19.3? 
Should we follow **strictly**  the corresponding signatures of vs code in other cases?

Reply:
We should expose only what we use, if it is not used, then it should not be exposed.